frontend/src/App.js

```diff
@@ -3,18 +3,21 @@ import TopBar from "./components/TopBar";
 import MainPage from "./components/MainPage";
 import Footer from "./components/Footer";
 import { Switch, Route } from "react-router-dom";
-import MoviePageFilterByTitle from "./components/MoviePageFilterByTitle";
+import MovieSheet from "./components/MovieSheet";
 import "./App.css";
-import ListMovies from "./components/ListMovies";
+import ListMoviesByGenre from "./components/ListMoviesByGenre";
```

Comment:
Shouldn't it be something like `Movies` ?

Reply:
Hello, for us just "Movies" would bring more confusion than understanding.... it a real list of movies in range by genre. For us it's quite clear